doc/modules/cassandra/pages/new/virtualtables.adoc

```diff
@@ -290,6 +293,22 @@ SELECT total - progress AS remaining
 FROM system_views.sstable_tasks;
 ....
 
+=== CQL metrics Virtual Table
```

Comment:
also, it looks like it needs to be moved up.

Reply:
Looks like existing ones are not in order, moved anyhow